static/app/views/dashboards/widgetBuilder/components/sortBySelector.tsx

```diff
@@ -103,7 +103,6 @@ function WidgetBuilderSortBySelector() {
             <ResultsLimitSelector
               disabled={disableSortDirection && disableSort}
               name="resultsLimit"
-              menuPlacement="auto"
```

Comment:
Checked everywhere where `menuplacement="auto"` was removed and everything works as expected